css/css.go

```diff
@@ -1201,6 +1201,35 @@ func (c *cssMinifier) minifyProperty(prop Hash, values []Token) []Token {
 			values[0].Data = oneBytes
 			values[0].Ident = 0
 		}
+	case UnicodeRange:
+		for i, value := range values {
+			// Copy data so can mutate without having to undo.
+			data := make([]byte, len(value.Data))
+			copy(data, value.Data)
+
+			// If we have a range of exactly two parts.
+			if parts := bytes.Split(data, []byte{'-'}); len(parts) == 2 {
+				parts[0] = bytes.TrimPrefix(parts[0], []byte("U+"))
+
+				// And both parts are the same length.
+				if len(parts[0]) == len(parts[1]) {
+					// Replace matching trailing 0s & Fs with question marks.
+					for j := len(parts[0]) - 1; j >= 0; j-- {
+						if parts[0][j] == '0' && parts[1][j] == 'F' {
+							parts[0][j] = '?'
+							parts[1][j] = '?'
+						} else {
+							break
+						}
+					}
+
+					// If both parts now match we only need one.
+					if bytes.Equal(parts[0], parts[1]) {
```

Comment:
Put this check first: in reverse order check if the first part is a zero and the second part an `F`. If this is not the case, check that both bytes are equal, or the left side is a `0` or is shorter than the right side (e.g. `U+0-04FF => U+4??` as far as I can see).

If you put this check first before setting the `?`s, we can get rid of the `copy()`.